lib/daterangepicker.js

```diff
@@ -953,6 +950,16 @@
                 this.chosenLabel = this.container.find('.ranges li:last').addClass('active').html();
                 this.showCalendars();
             }
+            var container = $('.daterangepicker .ranges ul'),
+                scrollTo = $('.daterangepicker .ranges ul .active'),
+                scrollPosition = scrollTo.offset().top - container.offset().top + container.scrollTop();
+
+
+            if (scrollPosition > 145) {
+                container.scrollTop(
+                    scrollTo.offset().top - container.offset().top + container.scrollTop()
+                );
+            }
```

Comment:
when you click a range, it will re-init the whole plugin and it will not scroll you down to the selected range on the right panel, which felt weird. this fixes that.